README.md

```diff
@@ -285,7 +285,7 @@ cd vcpkg
 # build vcpkg (remove .bat on Linux/Mac)
 .\bootstrap-vcpkg.bat
 # install dependencies (remove .exe in Linux/Mac) and update triplet
-vcpkg.exe install --triplet x64-windows-static curl[winssl] cmocka paho-mqtt
+.\vcpkg.exe install --triplet x64-windows-static curl[winssl] cmocka paho-mqtt

```

Comment:
Will this work outside of powershell as well?

Reply:
Yes, its valid in command prompts as well